ui/components/ImageAutomation/repositories/ImagePolicy.tsx

```diff
@@ -14,9 +15,10 @@ type Props = {
   namespace: string;
   clusterName: string;
 };
-function getInfoItems(data: any): InfoField[] {
-  const [imgPolicy] = Object.keys(data.obj.spec.policy);
-  const val = Object.values(data.obj.spec.policy[imgPolicy]);
+
+function getInfoItems(data: FluxObject): InfoField[] {
+  const [imgPolicy] = _.keys(_.get(data, ["obj", "spec", "policy"]));
+  const val = _.values(_.get(data, ["obj", "spec", "policy", imgPolicy]));
```

Comment:
@yiannistri An example of `lodash` handling `undefined` better than native JS APIs.